src/grpc_requests/aio.py

```diff
@@ -468,7 +479,14 @@ def _is_descriptor_registered(self, filename):
         except KeyError:
             return False
 
-    async def _register_file_descriptor(self, file_descriptor):
+    # In practice it always seems like descriptors are returned in an order that makes sense for dependency
+    # registration, but i can't find a guarantee in the spec
+    # Because of this, go one by one and register, using the other returned descriptors as possible dependencies
```

Comment:
```suggestion
    # Iterate over descriptors for registration, including returned descriptors as possible dependencies.
    # This is necessary as while in practice descriptors appear to be returned in an order that works for dependency
    # registration, this is not guaranteed in the reflection specification.
```

Just some suggested wordsmithing - I'm making the assumption the referred to spec is the GRPC reflection spec.